app/src/main/java/com/orange/ods/app/ui/components/listitem/ListItemCustomizationState.kt

```diff
@@ -28,28 +26,22 @@ import com.orange.ods.compose.component.listitem.OdsListItem
 @Composable
 fun rememberListItemCustomizationState(
     bottomSheetScaffoldState: BottomSheetScaffoldState = rememberBottomSheetScaffoldState(),
-    lineCount: MutableIntState = rememberSaveable { mutableIntStateOf(ListItemCustomizationState.DefaultLineCount) },
+    secondaryTextLineCount: MutableState<OdsListItem.SecondaryTextLineCount?> = rememberSaveable { mutableStateOf(OdsListItem.SecondaryTextLineCount.One) },
     selectedLeadingIconType: MutableState<OdsListItem.LeadingIcon.Type?> = rememberSaveable { mutableStateOf(null) },
     selectedTrailing: MutableState<Class<out OdsListItem.Trailing>?> = rememberSaveable { mutableStateOf(null) },
-) = remember(lineCount) {
-    ListItemCustomizationState(bottomSheetScaffoldState, lineCount, selectedLeadingIconType, selectedTrailing)
+) = remember(secondaryTextLineCount) {
```

Comment:
This is not related to the modifications in this PR, but I think we should write this instead:

```kotlin
remember(bottomSheetScaffoldState, secondaryTextLineCount, selectedLeadingIconType, selectedTrailing)
```